R/accountPageTrigger.R

```diff
@@ -0,0 +1,25 @@
+#'
+#' accountPageTrigger
+#'
+#' trigger event to redirect user to Account (payments) page,
+#' typically used in an `observeEvent`
+#'
+#' @importFrom shiny getDefaultReactiveDomain updateQueryString
+#'
+#' @param session A Shiny session object.
+#'
+#' @export
+#'
+
+accountPageTrigger <- function(
+  session = shiny::getDefaultReactiveDomain()
+) {
+
+  shiny::updateQueryString(
+    queryString = "?page=test",
+    session = session,
+    mode = "replace"
```

Comment:
The mode should be "push" so that we can use the browser's forward and back buttons.  Please test to make sure the forward and back buttons work with "push".